zkevm-circuits/src/bytecode_circuit/circuit.rs

```diff
@@ -683,6 +693,20 @@ impl<F: Field> BytecodeCircuitConfig<F> {
         Ok(())
     }
 
+    fn annotate_circuit(&self, region: &mut Region<F>) {
+        self.bytecode_table.annotate_columns_in_region(region);
+        self.keccak_table.annotate_columns_in_region(region);
+
+        region.name_column(|| "BYTECODE_q_enable", self.q_enable);
+        region.name_column(|| "BYTECODE_q_first", self.q_first);
+        region.name_column(|| "BYTECODE_q_last", self.q_last);
+        region.name_column(|| "BYTECODE_length", self.length);
+        region.name_column(|| "BYTECODE_push_data_left", self.push_data_left);
+        region.name_column(|| "BYTECODE_push_data_size", self.push_data_size);
+        region.name_column(|| "BYTECODE_value_rlc", self.value_rlc);
+        region.name_column(|| "BYTECODE_push_data_left_inv", self.push_data_left_inv);
```

Comment:
Now that the `IsZeroConfig` has annotation functions, you can just call them here :)
```rust
self.push_data_left_is_zero.annotate_columns_in_region(region, "BYTECODE");
```